plugins/woocommerce/tests/php/src/Internal/Admin/Settings/Mocks/FakePaymentGateway.php

```diff
@@ -8,12 +8,70 @@
  */
 class FakePaymentGateway extends \WC_Payment_Gateway {
 	/**
-	 * @var string Gateway ID.
+	 * Gateway ID.
+	 *
+	 * @var string
 	 */
 	public $id = 'fake-gateway-id';
 
 	/**
-	 * @var string Corresponding gateway plugin slug.
+	 * Gateway title.
+	 *
+	 * @var string
+	 */
+	public $title = 'Fake Gateway Title';
+
+	/**
+	 * Gateway description.
+	 *
+	 * @var string
+	 */
+	public $description = 'Fake Gateway Description';
+
+	/**
+	 * Gateway method title.
+	 *
+	 * @var string
+	 */
+	public $method_title = 'Fake Gateway Method Title';
+
+	/**
+	 * Gateway method description.
+	 *
+	 * @var string
+	 */
+	public $method_description = 'Fake Gateway Method Description';
+
+	/**
+	 * Corresponding gateway plugin slug.
+	 *
+	 * @var string
 	 */
 	public string $plugin_slug = 'fake-plugin-slug';
+
+	/**
+	 * The recommended payment methods list.
+	 *
+	 * @var array
+	 */
+	public array $recommended_payment_methods = array();
+
+	/**
+	 * Get the recommended payment methods list.
+	 *
+	 * @param string $country_code Optional. The business location country code. Provide a 2-letter ISO country code.
```

Comment:
We could potentially simply add a reference to the original `get_recommended_payment_methods` method instead of duplicating the doc comments so we don't have two places to maintain.

Reply:
The thing is, the `get_recommended_payment_methods` method is not part of the `WC_Payment_Gateway` API at this point. It is only implemented by the WooPayments gateway class.

I will reduce the comment to reduce the maintenance burden.